### PR TITLE
copyheart license added

### PR DESCRIPTION
**Pull request by starlord1311**
added copyheart license. similar to the pull request link that was sent

### CODE REVIEW

File: src/licensedcode/data/rules/copyheart_syntax_highlighter.yml.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+licenses:
+    - copyheart
+
+license_choice: yes
```
Comment (pombredanne) [inline src/licensedcode/data/rules/copyheart_syntax_highlighter.yml.txt]: This cannot be choice, there is only one ;). You can remove this line

File: tests/licensedcode/data/licenses/copyheart.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+license:
+    - Copyheart
+license_choice: yes
```
Comment (pombredanne) [inline tests/licensedcode/data/licenses/copyheart.yml]: Same as above: not a choice. Delete this

**Comment by starlord1311** [comment]
@pombredanne i did something wrong rn. can you explain where did i go wrong. i am not getting adding licenses properly

**Comment by pombredanne** [comment]
@starlord1311 You need to put only the text for the copyheart license in the .LICENSE file, and certainly not mojibake.

**Pull request closed without merging**